### PR TITLE
c16 translation update

### PR DESCRIPTION
**Pull request by ManlyMarco**
Thanks to @ScrewThisNoise for providing the translations.

### CODE REVIEW

**Review by GeBo1 — CHANGES_REQUESTED** [review]
Needs minor fixes around stray characters

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c16/00/80/translation.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel.”
+「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel./”
```
[the remark this answers is not in the document]
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c16/00/80/translation.txt:4]:
```suggestion
「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel.”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c16/20/105/translation.txt [hunk 2]
```diff
@@ -10,7 +10,7 @@
 「普通はそれでいいのよ。変に理屈で考えないで、感じたままに心が動くから、それが感動って言うのだし」=“That's better to begin with though. Being moved is all about just forgetting the logic and letting your heart move the way it wants to, afterall.”
 「……って、これじゃ結局理屈っぽいわね。さっきから私、何回『理屈』って言ってるのかしら、フフッ」=“...Wuh, there I go again. I wonder how many times I've used the word "logic" in the past five minutes, fufuh--”
 「はは、数えてはいなかったけど。かなり言ってたかもね。でも、俺はそういう[H名]だから、好きになったんだと思う」=“Hahah, I wasn't really keeping count. Probably a fair bit though. But, I do think that's why I fell in love with you.”
-「ぇっ、ちょっと……いきなり何を……フフッ、夜景のムードにあてられたのかしら？」=“Eh--, hey... where did that... fufuh--, did the mood get to you?”
+「ぇっ、ちょっと……いきなり何を……フフッ、夜景のムードにあてられたのかしら？」=“Eh--, hey...  where did that... fufuh--, did the mood get to you?”
```
[the remark this answers is not in the document]
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c16/20/105/translation.txt:13]:
```suggestion
「ぇっ、ちょっと……いきなり何を……フフッ、夜景のムードにあてられたのかしら？」=“Eh--, hey... where did that... fufuh--, did the mood get to you?”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c16/25/80/translation.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel.”
+「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel./”
```
[the remark this answers is not in the document]
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c16/25/80/translation.txt:4]:
```suggestion
「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel.”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c16/30/80/translation.txt [hunk 4]
```diff
@@ -1,7 +1,7 @@
 //
 // Dumped for Koikatsu v5.1 (Koikatsu Party) by KKP_TextDump v1.4.6.0
 //
-「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel.”
+「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel./”
```
[the remark this answers is not in the document]
Comment (ManlyMarco) [reply translation/en/RedirectedResources/assets/abdata/adv/scenario/c16/30/80/translation.txt:4]:
```suggestion
「うぅ～～んっ！　凄く解放的ね。ここで食べたら何でも美味しそう」=“Nnnnnn~--! So liberating, isn't it? Just about anything would taste good if eaten here, I feel.”
```

**Review by GeBo1 — APPROVED** [review]

**Pull request merged**